README.md

```diff
@@ -96,6 +96,14 @@ Type: `Boolean`
 
 Default: `true`
 
+##### `options.events`
+
+An array of event names to listen for. Useful if you only need to watch specific events.
+
+Type: `Array`
```

Comment:
This should accept an array or string.  I don't like having to give an array with a single item because it's easy to transform in the code.